rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,174 @@
+        
+      - Add the "rfc-proposed" and the "Governance" labels. 
+
+  - Share a link to the pull request for community review on the HumanCellAtlas **#dcp** slack channel
+
+#### DCP PM:
+- Assign a **Shepherd** by completing the *Shepherd* section in the RFC template and pushing the update. The **Shepherd** guides the RFC and its **Author(s)** through the process.
+- If there is a *Scientific "guardrails"* section in the RFC, assign the **Science Reviewer** as a reviewer of the pull request and add the "science-review-required" label.
+
+#### Shepherd:
+- Include the RFC submission in *This week in DCP* status update
+
+### Reviewing RFC(s)
+
+#### Approvers:
+- May assign specific reviewers in the RFC pull request
+
+#### Reviewers:
+- Add feedback to the pull request comment thread
+
+#### Science Reviewer:
+- When the feedback from the Science review is addressed, replace the "science-review-required" label with "science-review-completed".
+
+#### Author: 
+- Revise the pull request in response to feedback and push new commits
+
+### Shepherding RFC(s)
+
+#### Shepherd:
+- Monitor the community review and ensure that issues are addressed by the **Author(s)** in a reasonable time period.
+
+- When all issues are addressed and any required Science reviews are complete, summarize the review discussion for the **Approvers** in a top-level summary comment in the pull request. Replace "rfc-proposed" with "rfc-final-review".
+
+- For software RFC(s), share a link to the pull request for approval on the HumanCellAtlas **#tech-architecture** slack channel. Add the RFC as an agenda item to the next *DCP Architecture* meeting.
+
+- For governance RFC(s), share a link to the pull request for approval on the HumanCellAtlas **#dcp-project-mgmt** slack channel. Add the RFC as an agenda item to the next *DCP PM* meeting.
+
+- Include the RFC final review in *This week in DCP* status update
+
+### Approving RFC(s)
+
+#### Approvers:
+
+- At the *DCP PM* or *DCP Architecture* meeting, review the **Shepherd** summary. If there is rough consensus to approve the RFC, replace "rfc-final-review" with "rfc-approved"
+
+#### Author(s):
+- Rename the RFC from `0000-my-feature.md` to `rfc####-my-feature.md` (with leading zeros) where `####` is the next available RFC number
+
+- Create a link between the approved RFC and its pull request by updating the *DCP PR* section in the RFC template and pushing the update.
+
+- Merge RFC pull request
+
+**NOTE**: *The existence of an approved software RFC does not indicate any particular priority or commitment to implement the RFC, nor is an approved RFC a green light to implement. Approved RFCs simply demonstrate community agreement on a technical decision. Product priorities are managed and implementations scheduled via the DCP PM planning process.*
+
+#### Shepherd:
+
+- Validate that the RFC pull request was successfully updated and merged
+- Include the RFC approval in *This week in DCP* status update
```

Comment:
The this week in the DCP is a nice concept but new, where did it come from and is someone going to take ownership?

Reply:
It's _borrowed_ from [_This Week in Rust_](https://this-week-in-rust.org/) and [_Last Week in Kubernetes_](http://lwkd.info/). It's a scenario for @jodihir and @matthewspeir and the *Communication* team charter.

Reply:
This sort of idea takes a lot of work. Before it goes into the approved document we need to ensure someone actually has time to do it. If no one will in the first instance then we should consider alternative solutions.

Reply:
Agreed. Much can also be simply automated - https://github.com/cmr/this-week-in-rust.

Reply:
I really the weekly status update idea. I think it has come up before. I don't know that we'll be able to implement it first, but it would be great to do eventually.

Reply:
+1 to the weekly update via slack or email.  That plus RFCs would be a very streamlined way to stay up to date on HCA without attending a ton of meetings